Source/Utils/WarpXMovingWindow.cpp

```diff
@@ -3,6 +3,33 @@
 
 using namespace amrex;
 
+namespace {
+WarpXParser makeParser (std::string const& parse_function)
```

Comment:
Is this copy-paste from the other `WarpXParser makeParser (std::string const& parse_function)
Write` above? If so, could we merge them?

Reply:
Done. I have written the makeParser () function in WarpXUtil.H file.